File: Cmdlets/ThreeLetterWindowsLanguageName.vb
Public Class ThreeLetterWindowsLanguageName

    Public Name As String

    Public ReadOnly Property Count() As Integer
        Get
            Return Me.Cultures.Count
        End Get
    End Property

    Public ReadOnly Property NeutralsCount() As Integer
        Get
            Return Me.Cultures.Where(Function(C) C.IsNeutralCulture).Count()
        End Get
    End Property

    Public ReadOnly Property RegionalsCount() As Integer
        Get
            Dim Specifics = System.Globalization.CultureInfo.GetCultures(System.Globalization.CultureTypes.SpecificCultures)

            Return Me.Cultures.Where(Function(C) Specifics.Contains(C)).Count()
        End Get
    End Property

    Public Cultures As New System.Collections.Generic.List(Of System.Globalization.CultureInfo)

End Class

File: C#/Cmdlets/Cmdlets/ThreeLetterWindowsLanguageName.cs
using System.Data;
using System.Linq;

namespace TIKSN.Cmdlets
{
    public class ThreeLetterWindowsLanguageName
    {
        public System.Collections.Generic.List<System.Globalization.CultureInfo> Cultures = new System.Collections.Generic.List<System.Globalization.CultureInfo>();
        public string Name;

        public int Count
        {
            get { return this.Cultures.Count; }
        }

        public int NeutralsCount
        {
            get { return this.Cultures.Where(C => C.IsNeutralCulture).Count(); }
        }

        public int RegionalsCount
        {
            get
            {
                var Specifics = System.Globalization.CultureInfo.GetCultures(System.Globalization.CultureTypes.SpecificCultures);

                return this.Cultures.Where(C => Specifics.Contains(C)).Count();
            }
        }
    }
}